docs/hot-reloading.md

```diff
@@ -34,8 +34,8 @@ While UIx component tree can be reloaded in a traditional way, by re-rendering f
   {:devtools {:preloads [my.app.preload]}}}}
 ```
 
-5. That's it. Now you can edit UIx component, hit save and once reloaded in the browser, local state should be preserved.
+5. That's it. Now you can edit a UIx component, hit save, and once reloaded in the browser the local state should be preserved.
 
 ### Things to look out for
 
-Make sure that your UI is not re-rendered from the root in a trasitional way when using fast-refresh, otherwise fast-refresh won't work and you'll still get state reset.
+Make sure that your UI is not re-rendered from the root in a traditional way when using fast-refresh, otherwise fast-refresh won't work and the state will be reset.
```

Comment:
I assumed that "trasitional" was meant to be "traditional - hopefully that is correct.